powerline/segments/common/players.py

```diff
@@ -34,7 +34,7 @@ def _convert_seconds(seconds):
 
 
 class PlayerSegment(Segment):
-	def __call__(self, format='{state_symbol} {artist} - {title} ({total})', state_symbols=STATE_SYMBOLS, **kwargs):
+	def __call__(self, format='{state_symbol}|{artist} - {title}|({elapsed}/{total})', state_symbols=STATE_SYMBOLS, **kwargs):
```

Comment:
Pipes should be spaces back, this is more in line with powerline style (though having state/artist/title/(elapsed/total) in separate segments would be even more in line, but doing this properly would be far harder).